styles/styles.css

```diff
@@ -249,9 +254,42 @@ main > .section:first-of-type {
 }
 
 /* section metadata */
+
 main .section.light,
 main .section.highlight {
   background-color: var(--light-color);
   margin: 0;
   padding: 40px 0;
 }
+
+/* Day 1 (Part 2) Assignment #3 start */
+
+.light-grey{
+  background-color: var( --homepage-background-color);
+  height: 400px;
+  color: var(--link-color);
+  font-size: 24px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center; 
+}
```

Comment:
Class name and properties are not in sync. Create a different name instead of light-grey

Reply:
I have changed the CSS and classname.